app/lib/actions/katello/host/hypervisors_update.rb

```diff
@@ -42,6 +42,7 @@ def update_or_create_hypervisor(hypervisor_json)
           host.subscription_facet.uuid = hypervisor_json[:uuid]
           host.subscription_facet.import_database_attributes(host.subscription_facet.candlepin_consumer.consumer_attributes)
           host.subscription_facet.save!
+          host.subscription_facet.update_subscription_status('unsubscribed_hypervisor') if host.subscription_facet.candlepin_consumer.entitlements.empty?
```

Comment:
I think the if statement is unnecessary, as there isn't any way to regsiter a hypervisor with a subscription already attached.  Even if there was, just let the subscription_status figure that.  I guess I am proposing one of the two:
a) remove the 'if host.subscription_facet.candlepin_consumer.entitlements.empty?'
b) remove both the if piece and the forced status of 'unbusbscribe_hypervisor', so its just:  `host.subscription_facet.update_subscription_status`